infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/transport/sample/NettyLogReplicationClientChannelAdapter.java

```diff
@@ -57,29 +56,20 @@ public void stop() {
     }
 
     @Override
-    public void send(String endpoint, CorfuMessage msg) {
+    public void send(String nodeId, CorfuMessage msg) {
         // Check the connection future. If connected, continue with sending the message.
         // If timed out, return a exceptionally completed with the timeout.
-        if (channels.containsKey(endpoint)) {
-            log.info("Sending message to {} on cluster {}, type={}", endpoint, getRemoteClusterDescriptor().getClusterId(), msg.getType());
-            channels.get(endpoint).send(msg);
+        if (channels.containsKey(nodeId)) {
+            log.info("Sending message to node {} on cluster {}, type={}", nodeId, getRemoteClusterDescriptor().getClusterId(), msg.getType());
```

Comment:
Same here, keep endpoint info... to ease debugging... and wherever it applies..

Reply:
Done.